seeders/students.seeder.js

```diff
@@ -3,16 +3,132 @@ import Student from "../src/models/Student";
 
 const data = [
```

Comment:
I'm wondering if this data might be easier generated using a loop + for faker.

Also, do you think we need to include t-numbers? Since we already have record IDs, and students are defined as unique by first/last name, do we need to support this extra field?

Reply:
Currently the tnumber is being used as the reference argument  in assessment.seeder to identify studensts. While the students names are currently in the seeder are unique, I was keeping in mind that names are not gauranteed to be unique. Would you prefer that I remove the tnumber and make adjustments to resolve the tnumber reference?

Reply:
All real names are dropped. Fake data provided by faker.js. Tnumbers are gone.